src/service/database.ts

```diff
@@ -0,0 +1,12 @@
+import firebase from 'firebase'
```

Comment:
I get the same error here.

Reply:
fixed

Reply:
actually not fixed, if i change it i get a "TypeError: firebase_app__WEBPACK_IMPORTED_MODULE_0__.default.auth is undefined" error :/

Reply:
firebase/auth firebase/firestore also need to be imported separately.